File: data/src/main/java/com/arturogutierrez/openticator/storage/realm/model/CategoryRealm.java
package com.arturogutierrez.openticator.storage.realm.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class CategoryRealm extends RealmObject {

  @PrimaryKey
  private String categoryId;
  private String name;

  public String getCategoryId() {
    return categoryId;
  }

  public void setCategoryId(String categoryId) {
    this.categoryId = categoryId;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }
}

File: data/src/main/kotlin/com/arturogutierrez/openticator/storage/realm/model/CategoryRealm.kt
package com.arturogutierrez.openticator.storage.realm.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

@RealmModel
class CategoryRealm : RealmObject() {

  @PrimaryKey
  var categoryId: String = ""
  var name: String = ""
}
